README.md

```diff
@@ -45,7 +45,21 @@ It is up to the implementor of an EIP-2362 compliant contract to decide which `i
 
 These `id`s MUST however be uniform across all providers: multiple EIP-2362 compliant contracts can give different values and timestamps for the same `id`, but they must be referring to the same data point.
 	
-**TODO:** The specifics of how to generate or agree on the `id` of a data point have not been specified yet.
+Proposed Id generation fields: 
+A complet list is kept on Google doc [here](https://docs.google.com/spreadsheets/d/15TPUuPWrxh7eMGaJKIupOh6l5yG-abn8UJejVhHCWV8/edit?usp=sharing).
```

Comment:
```suggestion
A complete list is kept on Google doc [here](https://docs.google.com/spreadsheets/d/15TPUuPWrxh7eMGaJKIupOh6l5yG-abn8UJejVhHCWV8).
```

Reply:
@aesedepece  and @Amxx , Also, this list is also a proposal, feel free to add or edit. thanks!